<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CirclesDialog</class>
 <widget class="QWidget" name="CirclesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_localCircle">
       <property name="text">
        <string>Create Personal Circle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_extCircle">
       <property name="text">
        <string>Create External Circle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_editCircle">
       <property name="text">
        <string>Edit Circle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_refresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>378</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Showing details: </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Membership</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTreeWidget" name="treeWidget_membership">
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>IDs</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Personal Circles</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Public Circles</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Peers</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTreeWidget" name="treeWidget_friends">
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Friends</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Friends of Friends</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Others</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Permissions</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="checkBox_anon">
          <property name="text">
           <string>Anon Transfers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_disc">
          <property name="text">
           <string>Discovery</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="treeWidget_category">
          <column>
           <property name="text">
            <string>Share Category</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Circles_images.qrc"/>
 </resources>
 <connections/>
</ui>
